wp-content/civi-extensions/goonjcustom/Civi/DroppingCenterService.php

```diff
@@ -30,71 +30,51 @@ public static function droppingCenterTabset($tabsetName, &$tabs, $context) {
-      "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fdonation-box-list",
-    );
-
-    $logisticsCoordinationUrl = \CRM_Utils_System::url(
-      "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fdropping-center%2Flogistics-coordination",
-    );
-
-    $outcome = \CRM_Utils_System::url(
-      "wp-admin/admin.php?page=CiviCRM&q=civicrm%2Fdropping-center-outcome",
-    );
-
-    // Add the Status tab.
-    $tabs['status'] = [
-      'title' => ts('Status'),
-      'link' => $status,
-      'valid' => 1,
-      'active' => 1,
-      'current' => FALSE,
-    ];
-
-    // Add the Visit Details tab.
-    $tabs['visit details'] = [
-      'title' => ts('Visit Details'),
-      'link' => $visitDetails,
-      'valid' => 1,
-      'active' => 1,
-      'current' => FALSE,
-    ];
-
-    // Add the Donation Box/Register Tracking tab.
-    $tabs['donation tracking'] = [
-      'title' => ts('Donation Tracking'),
-      'link' => $donationTrackingUrl,
-      'valid' => 1,
-      'active' => 1,
-      'current' => FALSE,
+    $tabConfigs = [
+      'status' => [
+        'title' => ts('Status'),
+        'module' => 'afsearchDroppingCenterStatus',
+        'directive' => 'afsearch-dropping-center-status',
+        'template' => 'CRM/Goonjcustom/Tabs/CollectionCamp.tpl',
+      ],
+      'visitDetails' => [
+        'title' => ts('Visit Details'),
+        'module' => 'afsearchVisitDetails',
+        'directive' => 'afsearch-visit-details',
+        'template' => 'CRM/Goonjcustom/Tabs/CollectionCamp.tpl',
+      ],
+      'donationTracking' => [
+        'title' => ts('Donation Tracking'),
+        'module' => 'afsearchDroppingCenterDonationBoxRegisterList',
+        'directive' => 'afsearch-dropping-center-donation-box-register-list',
+        'template' => 'CRM/Goonjcustom/Tabs/CollectionCamp.tpl',
+      ],
+      'logisticsCoordination' => [
+        'title' => ts('Logistics Coordination'),
+        'module' => 'afsearchDroppingCenterLogisticsCoordination',
+        'directive' => 'afsearch-dropping-center-logistics-coordination',
+        'template' => 'CRM/Goonjcustom/Tabs/CollectionCamp.tpl',
+      ],
+      'outcome' => [
+        'title' => ts('Outcome'),
+        'module' => 'afformDroppingCenterOutcome',
+        'directive' => 'afform-dropping-center-outcome',
+        'template' => 'CRM/Goonjcustom/Tabs/CollectionCampService.tpl',
+      ],
     ];
 
-    // Add the Logistics Coordination tab.
-    $tabs['logistics coordination'] = [
-      'title' => ts('Logistics Coordination'),
-      'link' => $logisticsCoordinationUrl,
-      'valid' => 1,
-      'active' => 1,
-      'current' => FALSE,
-    ];
-
-    // Add the outcome tab.
-    $tabs['outcome'] = [
-      'title' => ts('Outcome'),
-      'link' => $outcome,
-      'valid' => 1,
-      'active' => 1,
-      'current' => FALSE,
-    ];
+    foreach ($tabConfigs as $key => $config) {
+      $tabs[$key] = [
+        'id' => $key,
+        'title' => $config['title'],
+        'is_active' => 1,
```

Comment:
The `is_active` property is not included in the tabConfigs array because it is a constant value that remains 1 for all tabs. this helps reduce redundancy in the code, as it eliminates the need to specify is_active for each tab configuration.